R/unnecessary_placeholder_linter.R

```diff
@@ -33,7 +33,7 @@
 #' @seealso [linters] for a complete list of linters available in lintr.
 #' @export
 unnecessary_placeholder_linter <- function() {
-  # TODO(michaelchirico): handle R4.2.0 native placeholder _ as well
+  # NB: Native placeholder '_' must be used with a named argument, so it's not relevant here.
```

Comment:
Sporadic diff?

Reply:
no, you merged #2495 into this PR 🙈

Reply:
Sorry for the confusing choice of merge target, but I was using the codespace to get an R4.3.0+ version running, so it had to include all the changes here and it was easier to just diff against this branch than to remove all the changes.

I split off the change to #2499 now.